optimum/habana/trl/trainer/ppo_trainer.py

```diff
@@ -820,7 +833,18 @@ def train_minibatch(
             old_logprobs, values, logits, vpreds, logprobs, mask, advantages, returns
         )
         loss = loss_p + loss_v
-        self.accelerator.backward(loss)
+        global _recorded_graph
+
+        if _recorded_graph is None:
+            _recorded_graph = ht.hpu.HPUGraph()
+            s = ht.hpu.default_stream()
+
+            with ht.hpu.stream(s):
+                _recorded_graph.capture_begin()
+                self.accelerator.backward(loss)
+                _recorded_graph.capture_end()
+        else:
+            _recorded_graph.replay()
```

Comment:
Out of curiosity, would that make sense to have the same logic in the `GaudiTrainer`?